controlplane/pkg/nsm/nsm.go

```diff
@@ -354,13 +354,13 @@ func (srv *networkServiceManager) getConnectionParameters(xcon *crossconnect.Cro
 		switch mm.GetType() {
 		case vxlan.MECHANISM:
 			m := vxlan.ToMechanism(mm)
-			srcIP, err := m.SrcIP()
-			dstIP, err2 := m.DstIP()
+			srcIP := m.SrcIP()
+			dstIP := m.DstIP()
 			vni, err3 := m.VNI()
```

Comment:
As other errors are there removed, maybe this one should be simply called `err`?